client/visualizer/src/main/sidebar.ts

```diff
@@ -125,8 +129,13 @@ export default class Sidebar {
     <b class="blue">Keyboard Shortcuts</b><br>
     LEFT - Step Back One Turn<br>
     RIGHT - Step Forward One Turn<br>
+<<<<<<< HEAD
```

Comment:
let's change this

Reply:
Whoops, forgot that merge conflict, can fix in ~2 hours